csharp-weatherforecast/README.md

```diff
@@ -4,19 +4,24 @@ A sample accelerator for C#.
 
 This sample is the Weather Forecast RESTful API application made available from Microsoft.
 
+The application also uses [Steeltoe](https://steeltoe.io) to provide dynamic logging and application management endpoints.
+
 The starting source for this sample was created using:
+
 ```
-$ dotnet new webapi --framework net6.0 --language C#
+dotnet new steeltoe-webapi --logging-dynamic-logger --management-endpoints
```

Comment:
Would it be helpful to add instructions on adding steeltoetemplates since they are not installed by default?

Reply:
hm... I think this instruction is actually a lie, this app uses the newer minimal api format, but the Steeltoe template doesn't